backend/src/test/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/SpecificationControllerUnitTest.kt

```diff
@@ -6,17 +6,16 @@ import com.nhaarman.mockitokotlin2.verify
 import com.tngtech.apicenter.backend.connector.rest.dto.SpecificationDto
 import com.tngtech.apicenter.backend.connector.rest.mapper.SpecificationFileDtoMapper
 import com.tngtech.apicenter.backend.domain.entity.*
-import com.tngtech.apicenter.backend.domain.service.SpecificationPersistor
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
 
 internal class SpecificationControllerUnitTest {
 
-    private val specificationPersistor: SpecificationPersistor = mock()
+    private val specificationHandler: com.tngtech.apicenter.backend.domain.handler.SpecificationHandler = mock()
```

Comment:
Why the fully qualified import? Do we have more than one `SpecificationHandler` class?